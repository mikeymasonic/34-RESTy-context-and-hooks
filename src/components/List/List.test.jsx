import React from 'react';
import { shallow } from 'enzyme';
import List from './List';

describe('List component', () => {
  it('renders List', () => {
    const wrapper = shallow(<List 
      requests={[]}
      handleClear={() => {}}
      handleLoad={() => {}}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
