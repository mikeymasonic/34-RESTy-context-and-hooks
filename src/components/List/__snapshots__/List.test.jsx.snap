// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component renders List 1`] = `
<div
  className="List"
>
  <ul />
  <button
    onClick={[Function]}
  >
    Clear
  </button>
</div>
`;
